### PR TITLE
enabled webpack html plugin

### PR DESCRIPTION
**Pull request by samkcarlile**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-// const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```
Comment (Shreshth3) [inline webpack.config.js:3]: fasdf

**Review by Shreshth3 — APPROVED** [review]
good

**Review by andrew-lovato — COMMENTED** [review]
Looks good

**Review by Jbaren01 — APPROVED** [review]
Great job

**Pull request merged**